### PR TITLE
Modernized LottieView

### PR DESCRIPTION
**Pull request by OguzYuuksel**
This commit gives possibility to access full configuration of LottiAnimationView from SwiftUI, please checkout inline documentation.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -3,13 +3,13 @@ import PackageDescription
 
 let package = Package(
     name: "LottieExtensions",
-    platforms: [.macOS(.v10_15), .iOS(.v13), .tvOS(.v13), .watchOS(.v6)],
```
Comment (melle) [inline Package.swift:6]: do we have to remove all other platforms? I'm not aware but there might be external users of this library  too.

File: Sources/LottieExtensions/LottieView.swift [hunk 2]
```diff
@@ -1,53 +1,132 @@
...
+    }
+}
+
+// MARK: - Helpers
+extension LottieView {
+    static private func animationView(_ wrapperView: UIView) -> LottieAnimationView {
+        guard let animationView = wrapperView.subviews.first as? LottieAnimationView
+        else {
+            preconditionFailure(
+                """
```
Comment (melle) [inline Sources/LottieExtensions/LottieView.swift]: Question of taste, but having multiline strings for a single line is a bit ... wasteful? Everything would easily fit in one line.

File: Package.swift (hunk 1 above)
Comment (OguzYuuksel) [reply Package.swift:6]: Building will fail in other platforms there were no compiler check in current version. Therefore, people cannot be affected by the change.

File: Sources/LottieExtensions/LottieView.swift (hunk 2 above)
Comment (OguzYuuksel) [reply Sources/LottieExtensions/LottieView.swift]: Right, probably forgot while changing string.

**Pull request merged**